src/Accordion/accordion.js

```diff
@@ -1,27 +1,29 @@
 import React, { useState } from 'react';
 
-const Accordion = ({ title }) => {
+const Accordion = () => {
+  const mystyle = {
+    color: 'white',
+    height: 40,
+    width: 80,
+    marginLeft: 1350,
+    backgroundColor: 'green',
+    borderRadius: 12,
+  }
+  
+  const [isActive, setIsActive] = useState(1);
 
-    const mystyle = {
-        color: 'white',
-        height: 50,
-        width: 80,
-        backgroundColor: 'green',
-        borderRadius: 12,
-      }
-
-  const [isActive, setIsActive] = useState(false);
+  const openSurveyBox = (id) => {
+      setIsActive(id)
+      
+    }
 
   return (
     <div className="accordion-item">
 
-      <div className="accordion-title" onClick={() => setIsActive(!isActive)}>
-        <div>{title}</div>
-        <div>{isActive ? <button style={mystyle} >Save</button>  : <button style={mystyle} >Saved</button> }</div>
-      </div>
-      {isActive && <div className="accordion-content">
-        
-        <div className="Servey1" id='tabid1'  >
+      <div className='survey-1' onClick={() => openSurveyBox(1)}>Survey 1 <button style={mystyle}>Save</button></div>
+      {isActive === 1 ? 
```

Comment:
@SagarAIMS Rather than writing all the accordion static, use an array of accordion(which will contain all the questions, answer points of the question, status of the accordion) and use [map](https://reactjs.org/docs/lists-and-keys.html#keys-must-only-be-unique-among-siblings) function to render all accordion. 

Note - Check the link attached in the above message.

Reply:
@talapallikaran  Sir, 
I have Create new accordion with dynamic method using array and map function